android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/NpsAnswer.kt

```diff
@@ -0,0 +1,96 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+
+@Composable
+fun NpsAnswer(
+    currentIndex: Int = -1,
```

Comment:
We will need this one, as the [pager](https://google.github.io/accompanist/pager/) is a lazy pager, which means if we don't set the current value, it will disappear.

Moreover, I changed the logic in [this PR](https://github.com/suho/kmm-ic/pull/123) for more consistent between answer views, please ignore this one.